docs/installation/configuration/cmake_options.rst

```diff
@@ -0,0 +1,173 @@
+These libraries can also be configured using Fast DDS CMake options.
+
+.. list-table::
+    :header-rows: 1
+
+    *   - Option
+        - Description
+        - Default
+    *   - :class:`THIRDPARTY`
+        - Activates the use of Fast CDR and the internal third-party libraries. It is set to ``ON`` |br|
+          if EPROSIMA_BUILD is ``ON``.
+        - ``OFF``
+    *   - :class:`THIRDPARTY_UPDATE`
+        - Activates the automatic update of Fast CDR and the internal third-party libraries.
+        - ``ON``
+    *   - :class:`THIRDPARTY_fastcdr`
+        - Activates the use of Fast CDR library.
+        - ``OFF``
+    *   - :class:`THIRDPARTY_Asio`
+        - Activates the use of Asio internal third-party library.
+        - ``OFF``
+    *   - :class:`THIRDPARTY_TinyXML2`
+        - Activates the use of TinyXML2 internal third-party library.
+        - ``OFF``
+    *   - :class:`THIRDPARTY_android-ifaddrs`
+        - Activates the use of android-ifaddrs, an implementation of :func:`getifaddrs` for Android. |br|
+          Only if :class:`ANDROID` is set to ``ON``.
+        - ``OFF``
+
+Test options
+------------
+
+*eProsima Fast DDS* comes with a full set of tests to check the performance of the library.
+These tests are divided according to the layer being tested; thus there are DDS-layer tests and RTPS-layer tests.
+The building and execution of these tests is specified by the Fast DDS CMake options shown in the table below.
+
+.. list-table::
+    :header-rows: 1
+
+    *   - Option
+        - Description
+        - Default
+    *   - :class:`FASTRTPS_API_TESTS`
+        - Enables the execution of black-box tests for Fast DDS using the Fast DDS RTPS-layer API.
+        - ``OFF``
+    *   - :class:`FASTDDS_PIM_API_TESTS`
+        - Enables the execution of black-box tests for Fast DDS using the Fast DDS DDS-layer API.
+        - ``OFF``
+
+Fast DDS tests are implemented using the GoogleTest framework.
+To run the tests it is required to enable this framework using the :class:`GTEST_INDIVIDUAL` option.
+The configuration of the installation directories is also allowed as indicated in the following table.
+
+.. list-table::
+    :header-rows: 1
+
+    *   - Option
+        - Description
+        - Default
+    *   - :class:`GTEST_INDIVIDUAL`
+        - Activate the execution of GoogleTest tests.
+        - ``OFF``
+    *   - :class:`EPROSIMA_GTEST`
+        - Activates special set of GTEST_ROOT (the root directory of the GoogleTest installation).
+        - ``OFF``
+    *   - :class:`EPROSIMA_GMOCK`
+        - Activates special set of GMOCK_ROOT (the root directory of the GoogleMock installation).
+        - ``OFF``
+
+Furthermore, several CMake options are available to specify which tests are to be executed.
+
+.. list-table::
+    :header-rows: 1
+
+    *   - Option
+        - Description
+        - Default
+    *   - :class:`PERFORMANCE_TESTS`
+        - Activates the building and execution of performance tests.
```

Comment:
Except for the video one. For that one, you need to set both `PERFORMANCE_TESTS=ON` and `VIDEO_TESTS=ON`